addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -484,7 +517,6 @@ export class SearchBar extends Component {
      */
     onItemMousemove(focusedIndex) {
         this.state.focusedIndex = focusedIndex;
-        this.inputRef.el.focus();
```

Comment:
I didn't see that. Why is it there?

Reply:
@Polymorphe57 Good catch, I forgot to remove that diff. 
It's now fixed

Reply:
and rebased